src/activities/activity.types.ts

```diff
@@ -1,10 +1,10 @@
-import { DataSpeedAvg } from '../data/data.speed-avg';
-import { DataPaceAvg } from '../data/data.pace-avg';
-import { DataSwimPaceAvg } from '../data/data.swim-pace-avg';
-import { DataPace } from '../data/data.pace';
-import { DataSpeed } from '../data/data.speed';
-import { DataSwimPace } from '../data/data.swim-pace';
-import { DataVerticalSpeedAvg } from '../data/data.vertical-speed-avg';
+import {DataSpeedAvg} from '../data/data.speed-avg';
```

Comment:
Off-topic with this PR: We should statute if braces have spaces or not between imports. Which IDE are you using? We must force it into the tslint.json. (I'm using webstorm on my side)

Reply:
Webstorm as well :-D 

I see the issue. I ll take a look why this....  

I don't mind if they should or not personally :-) I leave this up to you :-D